assessment.rb

```diff
@@ -1,12 +1,36 @@
 require 'csv'
 # Assessment answers to:
+rails g model Licensee #added properties in migrate file before running rails db:migrate
 
 # question 2:
-
+licensee = Licensee.new surname: 'Bishop',given_name: 'Caren',gender: 'f',height: 63, weight: 132, dob: '1943-09-26', eye_color: 'Brown', hair_color: 'Black'
```

Comment:
Almost. This will add values to the model, but not save them to the database. You need to either call `.save`, or have called `create` in the first place.